test/e2e/node/crictl.go

```diff
@@ -53,22 +53,22 @@ var _ = SIGDescribe("crictl", func() {
 		}
 
 		for _, testCase := range testCases {
-			// Choose an arbitrary node to test.
-			host := hosts[0]
-			ginkgo.By(fmt.Sprintf("SSH'ing to node %q to run %q", host, testCase.cmd))
+			for _, host := range hosts {
+				ginkgo.By(fmt.Sprintf("SSH'ing to node %q to run %q", host, testCase.cmd))
```

Comment:
When no hosts are found with SSH, this test will silently not do anything. Would it be better to test for this scenario and then skip the test?

Reply:
Yes, changed as suggested :+1: